Formula/mariadb.rb

```diff
@@ -10,6 +10,15 @@ class Mariadb < Formula
     sha256 "fa09ca1ec1a6557099eafe2b0955a78ec4841a795194400b321914f84aab99fe" => :mavericks
   end
 
+  devel do
+    url "http://ftp.osuosl.org/pub/mariadb/mariadb-10.2.1/source/mariadb-10.2.1.tar.gz"
+    sha256 "90b7a17f3372c92c12dff084b37fcca8c4cf8106f4dcabd35fadc8efbaa348a2"
+
+    # upstream fix for compilation error
```

Comment:
Alright, cool. Thanks for checking. Just didn't want to be carrying this around for yonks 😅.